### PR TITLE
chore(payloads): add code scanning examples

### PR DESCRIPTION
**Pull request by pholleran**
This PR adds payload examples for the following code scanning webhooks:

* `fixed`
* `closed_by_user`
* `created`

### CODE REVIEW

**Comment by gr2m** [comment]
The schema needs to be updated. The `"enterprise"` property is currently not defined for the `code_scanning_alert` event

```
  {
    keyword: 'additionalProperties',
    dataPath: '',
    schemaPath: '#/additionalProperties',
    params: { additionalProperty: 'enterprise' },
    message: 'should NOT have additional properties'
  }
```

**Comment by wolfy1339** [comment]
We only have examples from GitHub, and not GHES, which is why the `"enterprise"` property isn't defined in the schemas.

I think the `"enterprise"` property should be dropped for now and we can add it to all the schemas in a follow up issue

File: payload-examples/api.github.com/code_scanning_alert/created.payload.json [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+    "gists_url": "https://api.github.com/users/Codertocat/gists{/gist_id}",
+    "starred_url": "https://api.github.com/users/Codertocat/starred{/owner}{/repo}",
+    "subscriptions_url": "https://api.github.com/users/Codertocat/subscriptions",
+    "organizations_url": "https://api.github.com/users/Codertocat/orgs",
+    "repos_url": "https://api.github.com/users/Codertocat/repos",
+    "events_url": "https://api.github.com/users/Codertocat/events{/privacy}",
+    "received_events_url": "https://api.github.com/users/Codertocat/received_events",
+    "type": "User",
+    "site_admin": false
+  },
```
Comment (wolfy1339) [inline payload-examples/api.github.com/code_scanning_alert/created.payload.json:194]: This should be `github` as per the docs
> If the action is `reopened_by_user` or `closed_by_user`, the `sender` object will be the user that triggered the event. The sender object is `github` for all other actions.

File: payload-examples/api.github.com/code_scanning_alert/fixed.payload.json [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+    "gists_url": "https://api.github.com/users/Codertocat/gists{/gist_id}",
+    "starred_url": "https://api.github.com/users/Codertocat/starred{/owner}{/repo}",
+    "subscriptions_url": "https://api.github.com/users/Codertocat/subscriptions",
+    "organizations_url": "https://api.github.com/users/Codertocat/orgs",
+    "repos_url": "https://api.github.com/users/Codertocat/repos",
+    "events_url": "https://api.github.com/users/Codertocat/events{/privacy}",
+    "received_events_url": "https://api.github.com/users/Codertocat/received_events",
+    "type": "User",
+    "site_admin": false
+  },
```
Comment (wolfy1339) [inline payload-examples/api.github.com/code_scanning_alert/fixed.payload.json:213]: This should be `github` as per the docs
> If the action is `reopened_by_user` or `closed_by_user`, the `sender` object will be the user that triggered the event. The sender object is `github` for all other actions.

**Review by wolfy1339 — DISMISSED** [review]

File: payload-examples/api.github.com/code_scanning_alert/fixed.payload.json (hunk 2 above)
Comment (pholleran) [reply payload-examples/api.github.com/code_scanning_alert/fixed.payload.json]: Thanks for catching this

**Comment by wolfy1339** [comment]
All is good to go (mostly). Waiting on #379 before this can go any further

**Review by wolfy1339 — APPROVED** [review]

**Pull request merged**